packages/filesystem/src/node/node-filesystem.spec.ts

```diff
@@ -64,7 +67,10 @@ describe("NodeFileSystem", function () {
             expect(fs.statSync(FileUri.fsPath(uri_2)).isFile()).to.be.true;
 
             const stat = await fileSystem.getFileStat(root.toString());
-            expect(stat.children!.length).to.equal(2);
+            expect(stat).to.not.be.undefined;
+            if (stat) {
```

Comment:
Same for everywhere for the tests, just fail with a reference error. I think it is better than accidentally skipping an expectation. What do you think?